.gitignore

```diff
@@ -5,6 +5,7 @@
 .DS_Store
 Manifest.toml
 dev/
+.*.*.swp
```

Comment:
Does `*swp` work here instead?

Reply:
I just learned that I should be using a global configuration instead, so I will revert that change.
https://stackoverflow.com/questions/4824188/git-ignore-vim-temporary-files

Reply:
Oh I think it's fine to add this stuff to the .gitignore, I just thought `*swp` would catch it all and be simpler.